src/bscPlugin/hover/HoverProcessor.ts

```diff
@@ -39,6 +39,32 @@ export class HoverProcessor {
         return parts.join('\n');
     }
 
+    private buildCallableContents(callable: Callable, fence: (code: string) => string) {
+        const parts = [fence(callable.type.toString())];
+        
+        // For global callables, use shortDescription and documentation properties
+        if (callable.shortDescription || callable.documentation) {
+            const docs = [];
+            if (callable.shortDescription) {
+                docs.push(callable.shortDescription);
+            }
+            if (callable.documentation) {
+                docs.push(callable.documentation);
+            }
+            if (docs.length > 0) {
+                parts.push('***', docs.join('\n\n'));
+            }
+        } else {
+            // For regular callables, use token-based documentation
+            const tokenDocs = this.getTokenDocumentation((this.event.file as BrsFile).parser.tokens, callable.functionStatement?.func?.functionType);
+            if (tokenDocs) {
+                parts.push('***', tokenDocs);
+            }
+        }
+        
+        return parts.join('\n');
+    }
+
```

Comment:
Why did you have to write the code like this? Aren't the global callabales stored the same way as other functions? I'd rather have the documentation stored on the functions in a unoform way so that user-defined functions and global functions both expose their documentation in the same way. Go find where we expose the global callables and make sure their docs are included properly. Perhaps with comment tokens or something (just a thought, don't do this unless it's already being done this way elsewhere)